src/query/service/src/interpreters/interpreter_copy.rs

```diff
@@ -302,7 +306,67 @@ impl CopyInterpreter {
             .await?;
 
         let (mut build_res, source_schema, files) = if let Some(query) = &plan.query {
-            let (build_res, source_schema) = self.build_query(query).await?;
+            let mut build_res;
+            let (select_interpreter, source_schema) = self.build_query(query).await?;
+            let plan_query = select_interpreter.build_physical_plan().await?;
```

Comment:
If COPY is distributed, how we gather the status from the cluster through the coornidate query node?

Reply:
here is just a physical plan, we will use exchange physical plan at line 342